Robot_Adapter/Convert/ToRobot/Properties/Material.cs

```diff
@@ -48,10 +50,36 @@ public static void ToRobot(IRobotMaterialData materialData, IMaterialFragment ma
                 materialData.Kirchoff = isotropic.ShearModulus();
                 materialData.DumpCoef = isotropic.DampingRatio;
             }
+            else if (material is Timber)
+            {
+                Timber timber = material as Timber;
+                materialData.E = timber.YoungsModulus.X;
+                Engine.Reflection.Compute.RecordWarning("Young's modulus in Y and Z axis are ignored due to Robot definition limitation");
+                materialData.E_Trans = Math.Abs((timber.YoungsModulus.Y+timber.YoungsModulus.Z)/2);
+                Engine.Reflection.Compute.RecordWarning("Young transverse modulue (timber) is an average between Y and Z axis due to Robot definition limitation");
+                materialData.RO = timber.Density * Engine.Robot.Query.RobotGravityConstant;
+                materialData.Kirchoff = timber.ShearModulus.X;//Since shear modulus is expressed as shear stress over shear strain, longitudinal value is used as it is likely used for cross section analysis.
+                Engine.Reflection.Compute.RecordWarning("Shear modulus in Y and Z axis are ignored due to Robot definition limitation");
+                materialData.LX = timber.ThermalExpansionCoeff.X;//Value in X axis. Longitudinal expansion for bar element is likely of interets.
+                Engine.Reflection.Compute.RecordWarning("Thermal Expansion Coefficient in Y and Z axis are ignored due to Robot definition limitation");
+                materialData.NU = timber.PoissonsRatio.X;//Poisson ratio in X axis. Longitudinal defomation under axial loading is likely of interests.
```

Comment:
This can actually not be set through the UI in robot, so would completely skip setting this.

Reply:
I've deleted the PoissonsRatio for Timber material because as you mentioned it is not a component for Robot Timber material. I think thermalExpansionCoeff is steel worth keeping thought?